### PR TITLE
Added HelloWorld file.

### PR DESCRIPTION
**Pull request by 185driver**

### CODE REVIEW

**Comment by 185driver** [comment]
Added during Hacktoberfest event.

File: HelloWorld/helloworld.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+﻿<!DOCTYPE html>
+<html lang="en">
+
+<head>
```
Comment (t2013anurag) [inline HelloWorld/helloworld.html]: can you give 2 spaces indentation in tags

Comment (t2013anurag) [reply HelloWorld/helloworld.html]: also use hello_world directory don't create a new directory.

**Comment by 185driver** [comment]
Changes made.  Thanks.

**Pull request merged**